src/simplay-web/tester/events.json

```diff
@@ -0,0 +1,151 @@
+{
+  "events": [
+    {
+      "action": "setPosition",
```

Comment:
Just as a heads up, these actions here will be different, once implemented correctly, so don't rely on this file for testing :)